performance-firebase/build.gradle

```diff
@@ -49,11 +49,16 @@ android {
 dependencies {
     //Depends on Base Skeletoid
     api project(path: ':base')
-    testImplementation project(path: ':base', configuration: 'childTestDependencies')
+
+    implementation "androidx.core:core-ktx:1.3.0"
```

Comment:
Probably this can be removed. Android Studio when I updated it auto added this...